drinks/authSystem/tests/test_login.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth.models import User
+from django.test import Client
+from django.test.testcases import TestCase
+
+
+class TestLogin(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create_user(username='testuser', password='123')
+
+    def test_get(self):
+        response = self.client.get("/login/")
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'login_user.html')
+
+    def test_post(self):
```

Comment:
jak mamy test który testuje happy path to możemy też dać takie info w nazwie, że np `test_login_when_correct_credentials`